### ISSUE TITLE
Ability to use existing Actor images from MCM/MB with XBMC

### ISSUE THREAD

**Issue by psouza4** [psouza4/mediacentermaster#780]
Originally reported on Google Code with ID 780

```
I have a LOT of actor images that I got using MCM or from a zip file that I got from
a forum somewhere else, i would like MCM to be able to utilise those images when you
tell it to generate XBMC info.

Ideally a sym link in the Media folder for the artist pointing to the location set
in MCM & "artist_name" then they would all be in the same place like present and would
save duplications :P

Not sure if that would work, but it would be nice :D

Anyway the ability for MCM to get/use actor images with XBMC. 
```

Reported by `mad3813` on 2011-09-25 19:14:11

**Comment by psouza4** [comment]
Media Center Master will now store actor image URLs in the metadata so that they can be exported to Kodi/XBMC without the need to translate local paths.

**Comment by psouza4** [comment]
Released in version 2.18 build 4518.858 (2.18.4518.858).
